modules/microshift-storage-device-classes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-device-classes_{context}"]
+== Device classes
```

Comment:
```suggestion
= Device classes
```
See the comment above about using an H1 and using the level offset in the assembly.